tools/bt2img.py

```diff
@@ -54,18 +52,29 @@
     "Wait",
     "ClearEntireCostmap",
     "ReinitializeGlobalLocalization",
+    "Action"
     ]
 condition_nodes = [
     "IsStuck",
     "GoalReached",
     "initialPoseReceived",
-    "GoalUpdated",
```

Comment:
This shouldn't be deleted. It would be great if we could update this list with the newer condition nodes actually.

Reply:
Can you list them for him to add?

Reply:
DistanceTraveled, GoalReached, GoalUpdated, IsStuck, TimeExpired, TransformAvailable, InitialPoseReceived

Reply:
May I ask why you don‘t want to use the explicit way of describing bt nodes in xml?  

So writing `<Action ID=„FinishTaskNode“ />` instead of `<FinishTaskNode />` leads to automatic detection of the node type.   

Then no one will have to update this script again just for a new node being added to the xml.  

I use this script also for our internal nodes and there is no mention of their names in the script.

Reply:
I get that we won't have to update this script on adding new BT nodes if we go for the explicit way but defining the xml with just the node name seems easier in general I think. Plus, keeping the node names in these lists along with the node types will allow us to support both implicit and explicit way of writing xmls.